app.rb

```diff
@@ -0,0 +1,143 @@
+# rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
[Optional] I kindly suggest that we avoid suppressing linters warnings. They most times point towards writing a more efficient code.